GANDLF/utils/tensor.py

```diff
@@ -101,7 +108,9 @@ def reverse_one_hot(predmask_tensor: torch.Tensor, class_list: Union[List[int],
     for _class in class_list:
         for case in special_cases_to_check:
             if isinstance(_class, str):
-                if case in _class:  # check if any of the special cases are present
+                if (
+                    case in _class
+                ):  # check if any of the special cases are present
```

Comment:
Run `black` through this again.
```suggestion
                # check if any of the special cases are present
                if (
                    case in _class
                ):
```